drivers/hmis_external_apis/app/models/hmis_external_apis/ac_hmis/importers/housing_assessment_importer.rb

```diff
@@ -79,7 +79,8 @@ def complete_assessment(waitlist, assessment)
     tuples << ['housing_needs_homeless_duration', 'A year or more'] if waitlist.chronically_homeless == 'Yes' # infer to keep chronicity true when opened for editing
 
     # Note: Assessment is present in the export file, so we can assume the client is Eligible, Posted to the waitlist, and authorized.
-    tuples << ['housing_needs_result_type', 'Eligible'] # TODO: may need to add/change depending on #8129
+    tuples << ['housing_needs_result_type', 'Eligible']
+    tuples << ['housing_needs_eligible_project_types', 'PH_RRH']
```

Comment:
This is necessary to make the calculated fields display correctly in the form, it's not used anywhere else. I think there may be an underlying issue with autofill_readonly flag not being respected, but I opted to just make it happy rather than dig into that (for now)